### ISSUE TITLE
/fixusings reports single using option as "Ambiguous" for attribute

### ISSUE THREAD

**Issue by nickspoons** [OmniSharp/omnisharp-roslyn#2076]
Endpoint `/fixusings` is used by OmniSharp-vim (but not by OmniSharp-vscode, AFAICT). As well as removing redundant using statements, the endpoint finds missing using statements and when there is only a single possibility, adds it. When there are multiple possibilities, these are reported as "ambiguous" and the user needs to resolve their usings manually (or by selecting a code action).

Working example, where the element missing a using statement is a property (`Console`):

```cs
public class AClass
{
    public AClass()
    {
        Console.WriteLine("asdf");
    }
}
```

Running endpoint on this code returns the following response:

```json
{
  "Request_seq": 1041,
  "Command": "/fixusings",
  "Running": true,
  "Success": true,
  "Message": null,
  "Body": {
    "Buffer": null,
    "AmbiguousResults": [],
    "Changes": [
      {
        "NewText": "using System;\n\n",
        "StartLine": 1,
        "StartColumn": 1,
        "EndLine": 1,
        "EndColumn": 1
      }
    ]
  },
  "Seq": 821,
  "Type": "response"
}
```

However, take _this_ code, where the element missing a using statement is an attribute (`Obsolete`):

```cs
public class AClass
{
    [Obsolete]
    public AClass()
    {
    }
}
```

Now we get this response:

```json
{
  "Request_seq": 1053,
  "Command": "/fixusings",
  "Running": true,
  "Success": true,
  "Message": null,
  "Body": {
    "Buffer": null,
    "AmbiguousResults": [
      {
        "FileName": "C:\\code\\test\\AClass.cs",
        "Line": 3,
        "Column": 6,
        "EndLine": 1,
        "EndColumn": 1,
        "Text": "`Obsolete` is ambiguous. Namespaces: using System;",
        "Projects": []
      }
    ],
    "Changes": []
  },
  "Seq": 878,
  "Type": "response"
}
```

There is only a single possibility, but it is returned as an "ambiguous result", instead of a "change".

This is not a recent bug. I have tested back to 1.34.9 and this issue is present there. The issue occurs in both dotnet and .NET Framework projects, both in Windows and linux environments.
